packages/block/src/consensus/index.ts

```diff
@@ -0,0 +1,12 @@
+export {
+  CLIQUE_EXTRA_SEAL,
+  CLIQUE_EXTRA_VANITY,
+  cliqueEpochTransitionSigners,
+  cliqueExtraSeal,
+  cliqueExtraVanity,
+  cliqueIsEpochTransition,
+  cliqueSigHash,
+  cliqueSigner,
+  cliqueVerifySignature,
+} from './clique.js'
```

Comment:
The only function we don't want imported/used by users is (`_requireClique`)[https://github.com/ethereumjs/ethereumjs-monorepo/blob/41ffe667f6d5247f7e1ed39522bb22d6babb57f5/packages/block/src/consensus/clique.ts#L22]. Could export all functions, but this function too will be exported since it gets used in a separate file internal to the block package that would need to use it.

Reply:
Ok, makes sense, no big deal, than just leave it as it is.